.github/workflows/_build_base.yaml

```diff
@@ -57,10 +52,22 @@ permissions:
 
 jobs:
 
+  metadata:
```

Comment:
This logic should be placed directly in `ci.yaml` since it is a global condition. `_build_base.yaml` can take the output of the logic as an input.

Reply:
Moved to `ci.yaml`, as well as `BUILD_DATE`